modules/task/task.go

```diff
@@ -36,6 +36,7 @@ var (
 	_globalBackendMu sync.RWMutex
 	_globalBackend   Backend = &NopBackend{}
 	_asyncMod        service.Module
+	_asyncModErr     error
```

Comment:
Need it to be global so every time the function is called, it returns the module and error from the very first call.